<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student_Home</class>
 <widget class="QMainWindow" name="Student_Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Home</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="headerGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="headerHud">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>360</width>
       <height>140</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/header_large.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/back.png</normaloff>:/res/assets/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="headerWelcomeLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>240</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFF;
	font-family: Poppins;
	font-size: 20px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>Hello, &lt;?&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="dateLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <fontweight>Light</fontweight>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	text-align: right;
	font-family: Poppins;
	font-size: 9px;
	font-style: normal;
	font-weight: 300;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>Date&lt;?&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	text-align: right;
	font-family: Poppins;
	font-size: 11px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>Time&lt;?&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="avatarLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/cict_avatar.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="headerProgramLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	font-family: Poppins;
	font-size: 12px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>C&lt;?&gt; - P&lt;?&gt; YS&lt;?&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="headerStudentIdLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>85</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #15CAE3;
	font-family: Poppins;
	font-size: 12px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>20**M****</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="qrInfoGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>360</width>
      <height>420</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="presenceIcon">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>320</width>
       <height>320</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/presence_logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="qrcode">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>65</y>
       <width>180</width>
       <height>180</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="schoolLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>280</y>
       <width>240</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #000F2A;
	font-family: Poppins;
	font-size: 14px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>West Visayas State University</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>320</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #19283F;
	text-align: center;
	font-family: Poppins;
	font-size: 24px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>LN&lt;?&gt;, FN&lt;?&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="studentIdLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>320</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #19283F;
	font-family: Poppins;
	font-size: 18px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>20**M****</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="programLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>385</y>
       <width>320</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #19283F;
	font-family: Poppins;
	font-size: 18px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>C&lt;?&gt; - P&lt;?&gt; YS&lt;?&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="footerGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>360</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="footerHud">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>340</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/footer.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="homeButton">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>17</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/home_selected.png</normaloff>:/res/assets/home_selected.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="homeLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>42</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #15CAE3;
	text-align: center;
	font-family: Poppins;
	font-size: 6px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>qrInfoGroup</zorder>
   <zorder>headerGroup</zorder>
   <zorder>footerGroup</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
